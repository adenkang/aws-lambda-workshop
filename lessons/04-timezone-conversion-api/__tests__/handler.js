const { handler } = require('../src/handler')

test('It should convert timezone', () => {
  handler({
    pathParameters: {
      timestamp: '2018-05-17T22:13:10',
      sourceTimezone: 'Europe/London',
      targetTimezone: 'Australia/Sydney'
    }
  }, {}, (err, response) => {
    expect(err).toBe(null)
    expect(response).toMatchSnapshot()
  })
})
