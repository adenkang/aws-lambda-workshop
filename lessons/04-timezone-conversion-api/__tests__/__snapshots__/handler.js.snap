// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should convert timezone 1`] = `
Object {
  "body": "{\\"timestamp\\":\\"2018-05-18T07:13:10+10:00\\"}",
  "headers": Object {
    "Content-Type": "application/json",
  },
  "statusCode": 200,
}
`;
